### PR TITLE
Added Device_ID to expected data from SimpleReport

### PR DESCRIPTION
**Pull request by jimduff-usds**
This PR tweaks the expected PDI schema.

## Changes
- Add Device_ID as an expected field.
- Fixed "Street2" to be "Street_2" for consistency.
- Updated the example csv.

## Checklist
- [X] Tested locally?
- [ ] Added new dependencies?
- [X] Updated the docs?
- [ ] Added a test?
- [X] Did you check for sensitive data, and remove any?
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

## To Be Done

- Device_ID is just TEXT right now; probably need a controlled vocab for this.

### CODE REVIEW

File: prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema [hunk 1]
```diff
@@ -155,6 +155,11 @@ elements:
   - name: standard.specimen_type
     csvField: Specimen_type_code
 
+  # The type of device used for the test.  eg, BinaxNOW COVID-19 Ag Card
+  - name: standard.equipment_model_id
+    csvField: Device_ID
+
```
Comment (jimduff-usds) [inline prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema:161]: Mapping Device_ID to existing 'standard.equipment_model_id'.    It wasn't clear to me what SimpleReport should use for this field, so its just a vague string at the moment.   HHS "How to report Covid ELR" notes imply that this should map to Column "O" in the LIVD spreadsheet, but that column is empty for some common tests, eg Binax.

**Review by mry-usds — APPROVED** [review]
Looks good.

File: prime-router/docs/simplereport-data-dictionary.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
+# Data Dictionary for data incoming from SimpleReport
```
Comment (jimduff-usds) [inline prime-router/docs/simplereport-data-dictionary.md:1]: I'd much prefer to auto-generate this, but this took me just a few minutes to do it manually, so I'm throwing this out there for now.

File: prime-router/docs/getting_started.md [hunk 3]
```diff
@@ -46,6 +48,13 @@ You should be able to compile the project now. Check if it works.
...
 ```
 
+Check out the database if you like:
+```
+psql prime_data_hub
+    select * from task;
+```
+
+
 There are a few database commands that are helpful while developing.
```
Comment (RickHawesUSDS) [inline prime-router/docs/getting_started.md:58]: Nice.

File: prime-router/metadata/schemas/covid-19.schema [hunk 4]
```diff
@@ -284,9 +284,9 @@ elements:
     hhsGuidanceField: Ordering provider address
     hl7Field: ORC-24-1
 
-  - name: ordering_provider_street2
+  - name: ordering_provider_street_2
     type: STREET
-    natFlatFileField: Ordering_provider_street2
+    natFlatFileField: Ordering_provider_street_2
     hhsGuidanceField:
```
Comment (RickHawesUSDS) [inline prime-router/metadata/schemas/covid-19.schema:290]: National FlatFile has the field listed as `Ordering_provider_street2`. I'm looking at a file dated August 8, 2020.

**Review by RickHawesUSDS — CHANGES_REQUESTED** [review]
Looks good. Could you double check the change from `street2` to `street_2`. It doesn't match with the National flatfile that I have.

File: prime-router/metadata/schemas/covid-19.schema (hunk 4 above)
Comment (jimduff-usds) [reply prime-router/metadata/schemas/covid-19.schema:290]: Bleh.   I hunted around for that but couldn't find it.   We had a mixture of street2 and street_2; I was trying to be consistent.    Sigh.   I'll change it all back....

File: prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema [hunk 5]
```diff
@@ -124,8 +124,8 @@ elements:
   - name: standard.ordering_facility_street
     csvField: Ordering_facility_street
 
-  - name: standard.ordering_facility_street_2
-    csvField: Ordering_facility_street_2
+  - name: standard.ordering_facility_street2
+    csvField: Ordering_facility_street2
```
Comment (jimduff-usds) [inline prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema:128]: I'm never touching this street stuff again....

**Review by jimduff-usds — COMMENTED** [review]
Fixed per Rick.

**Pull request merged**